integrations/dynatrace/client.py

```diff
@@ -0,0 +1,104 @@
+from enum import StrEnum
+from typing import Any, AsyncGenerator
+
+import httpx
+from loguru import logger
+from port_ocean.context.event import event
+from port_ocean.utils import http_async_client
+
+
+class ResourceKey(StrEnum):
+    PROBLEM = "problem"
+    SLO = "slo"
+    ENTITY_TYPE = "entity_type"
+    ENTITY = "entity"
+
+
+class DynatraceClient:
+    def __init__(self, host_url: str, api_key: str) -> None:
+        self.host_url = host_url
+        self.client = http_async_client
+        self.client.headers = httpx.Headers({"Authorization": f"Api-Token {api_key}"})
```

Comment:
Our http_async_client has custom headers such as managing retry transport etc. So instead of overwriting it, you could instead update it using an example code copied from the Argocd integration

`        self.http_client.headers.update({"Authorization": f"Api-Token {api_key}"})`